lib/fluent/plugin/in_tail.rb

```diff
@@ -580,6 +586,29 @@ def on_notify
           @log.error_backtrace
         end
       end
+
+      class LineBufferFlusher
```

Comment:
`LineBufferIntervalFlusher` or `LineBufferTimerFlusher` looks more descriptive. If we say `LineBufferFlusher`, it must be responsible for all kinds of line buffer flushing, but this class is not.